### PR TITLE
Publish on Fleek Github workflow

### PR DESCRIPTION
**Pull request by chilcano**
## Description

* Adds Github workflow to publish content on Fleek based on branch.
* Adds Fleek API Key as secret
* Adds Fleek configuration

Task: [DOPS-595](https://aragonassociation.atlassian.net/browse/DOPS-595)

[DOPS-595]: https://aragonassociation.atlassian.net/browse/DOPS-595?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Michael-A-Heuer** [comment]
Is this really needed? See the discussion on discord: https://discord.com/channels/672466989217873929/1176842352044814408/1177333413213126737

**Review by Michael-A-Heuer — CHANGES_REQUESTED** [review]
Is this really needed? See the discussion on discord: https://discord.com/channels/672466989217873929/1176842352044814408/1177333413213126737

File: .github/workflows/publish-on-fleek.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      - name: Activate Fleek
+        run: mv fleek_${{ inputs.source_branch }}.json .fleek.json
+
+      - name: Deploy on Fleek
+        id: deploy
+        uses: fleekhq/action-deploy@v1
+        with:
+          apiKey: ${{ secrets.FLEEK_API_KEY }}
+
+      - name: Get Fleek outpu URL ( Branch=${{ inputs.source_branch }} )
```
Comment (Michael-A-Heuer) [inline .github/workflows/publish-on-fleek.yaml:44]:
```suggestion
      - name: Get Fleek output URL ( Branch=${{ inputs.source_branch }} )
```

File: fleek_master.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "source": "github",
+    "name": "sweet-butterfly-2931"
+  },
+  "build": {
+    "baseDir": "",
+    "publicDir": "build/",
+    "command": "yarn && yarn build",
+    "rootDir": ""
+  }
+}
```
Comment (Rekard0) [inline fleek_master.json:15]: add empty line at the the end

File: fleek_staging.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "source": "github",
+    "name": "holy-feather-8175"
+  },
+  "build": {
+    "baseDir": "",
+    "publicDir": "build/",
+    "command": "yarn && yarn build",
+    "rootDir": ""
+  }
+}
```
Comment (Rekard0) [inline fleek_staging.json:15]: add empty line at the the end

**Review by Michael-A-Heuer — APPROVED** [review]

**Pull request merged**